### PR TITLE
Feat/accept balsamic json

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds support for Balsamic orders as json

### How to prepare for test
- [x] ssh to clinical-db (depending on type of change)
- [x] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] login to OP-stage
- [x] do submit an Balsamic order as json 

### Expected test outcome
- [x] check that the order is accepted by the system
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by MM
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Deploy this branch
- [ ] Inform to Hero (customer), VW, lab, production

### CODE REVIEW

File: cg/meta/orders/status.py [hunk 1]
```diff
@@ -451,6 +451,7 @@ def store_rml(
             with self.status.session.no_autoflush:
                 application_version = self.status.current_application_version(pool["application"])
                 if application_version is None:
+                    print(f"Invalid application: {pool['application']}")
```
Comment (moonso) [inline cg/meta/orders/status.py:454]: Print?

**Review by moonso — APPROVED** [review]
👍

File: cg/meta/orders/status.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/meta/orders/status.py]: removed

**Comment by patrikgrenfeldt** [comment]
@Clinical-Genomics/laboratory @Clinical-Genomics/production @vwirta
Support for Balsamic-json has been deployed to Clinical-db:
image

**Pull request merged**